scripts/utils/exec.ts

```diff
@@ -20,27 +24,16 @@ export const exec = async (
   }
 
   logger.debug(command);
-  return new Promise((resolve, reject) => {
-    const defaultOptions: ExecOptions = {
-      silent: false,
-    };
-    const child = shell.exec(command, {
-      ...defaultOptions,
-      ...options,
-      async: true,
-      silent: false,
-    });
-
-    child.stderr.pipe(process.stderr);
+  const defaultOptions: Options = {
+    stdout: debug ? 'inherit' : 'ignore',
+  };
+  try {
+    await execa.command(command, { ...defaultOptions, ...options });
+  } catch (err) {
+    logger.error(chalk.red(`An error occurred while executing: \`${command}\``));
+    logger.log(errorMessage);
+    logger.log(err.message);
```

Comment:
Shouldn't we stop if there's an error executing a specific command?

Reply:
And likely the user wants to see the output too if it fails.

Reply:
Good point, I'll rethrow the error and remove that logger.error